inc/Validation/ConfigSchemas.php

```diff
@@ -108,6 +89,14 @@ private static function generate_remote_data_block_config_schema(): array {
 					] )
 				)
 			),
+			'overrides' => Types::nullable(
+				Types::list_of(
+					Types::object( [
+						'name' => Types::string(),
+						'display_name' => Types::nullable( Types::string() ),
```

Comment:
I don't know that we should allow this to be null. If we start from the beginning enforcing it to be there we don't ever have to think about it... and I imagine you really always should have it.

But if you think it makes sense to allow null I'm ok with it.

Reply:
Agreed, we can make it required